### PR TITLE
test(clean): add tests for clean_lat_long

### LINKED ISSUE #372: tests for clean_lat_long

### PR DESCRIPTION
**Pull request by brandonlockhart**
# Description

This PR adds simple tests for clean_lat_long. Closes #372

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have already squashed the commits and make the commit message conform to the project standard.
- [x] I have already marked the commit with "BREAKING CHANGE" or "Fixes #" if needed.
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by yxie66** [comment]
Hi, Brandon, great job! My comments:

1. For the naming of the functions, why not indicate the test coverage in function name, say name them to something like `test_split()` , similar with naming styles of EDA/Connector test functions?

2. For `test_9()`, in order to test `validate_lat_long()` with input of single instance, do you think it's necessary to include more typical test cases? Also I think some `assert` statement can be added to verify the output of `validate_lat_long()`, when input is single value.

File: dataprep/tests/clean/test_clean_lat_long.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+    df = pd.DataFrame(
+        {
+            "messy_lat": ["30° E", "41° 30′ N", "41 S", "80", "hello", "NA"],
+            "messy_long": ["30° E", "41° 30′ N", "41 W", "80", "hello", "NA"],
+        }
+    )
+    return df
+
+
+def test1(simple_df: pd.DataFrame) -> None:
```
Comment (yxie66) [inline dataprep/tests/clean/test_clean_lat_long.py:50]: Naming style can be more relevant to coverage of each test

**Comment by brandonlockhart** [comment]
Thanks Yi.

1. Okay, sounds good
2. 

>  do you think it's necessary to include more typical test cases?

Okay, I can include more

> Also I think some assert statement can be added to verify the output of validate_lat_long(), when input is single value.

Good idea. But I'm curious why you suggest verifying the result in this case and not all functions?

**Comment by yxie66** [comment]
> Good idea. But I'm curious why you suggest verifying the result in this case and not all functions?

From my understanding, for `validate_lat_long()` we care about its accurateness for single instance, while for `clean_url()` it's hard to verify output of each value. If we want to verify the result for `clean_url()`, maybe it's better to check the validness of output Dataframe. For example, the output Dataframe is not null and contains same number of rows as input. Do you have some suggestions? I didn't have a concrete idea yet.

**Comment by brandonlockhart** [comment]
In my opinion, if in these tests we don't want to only check that the function succeeds like in EDA and instead also also check correctness, then we should check that the output of each function is accurate.

**Comment by yxie66** [comment]
> In my opinion, if in these tests we don't want to only check that the function succeeds like in EDA and instead also also check correctness, then we should check that the output of each function is accurate.

I'm okay with either. If we are going to check whether output is accurate, I think desired output dataframe should be constructed for each function

**Comment by yxie66** [comment]
PR merged. Good job, thank you @brandonlockhart !

**Pull request merged**